src/main/java/com/spotify/github/v3/clients/RepositoryClient.java

```diff
@@ -358,6 +362,60 @@ public ChecksClient createChecksApiClient() {
         .thenAccept(IGNORE_RESPONSE_CONSUMER);
   }
 
+  /**
+   * Perform a merge.
+   *
+   * Both base and head must be branch names in the repo. To compare branches across other
+   * repositories in the same network as this repo, use the format <USERNAME>:branch.
```

Comment:
`<USERNAME>:branch` doesn't work, but you can pass a sha from a fork as `head`.
```suggestion
   * If base and head are branch names, they must be in this repo. To merge branches from
   * other repositories in the same network as this repo, use the sha.
```